yarn.lock

```diff
@@ -8952,6 +8952,92 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@o3r/workspace@workspace:packages/@o3r/workspace":
+  version: 0.0.0-use.local
+  resolution: "@o3r/workspace@workspace:packages/@o3r/workspace"
+  dependencies:
+    "@ama-sdk/schematics": "workspace:^"
+    "@angular-devkit/architect": ~0.1602.0
+    "@angular-devkit/build-angular": ~16.2.0
+    "@angular-devkit/core": ~16.2.0
+    "@angular-devkit/schematics": ~16.2.0
+    "@angular-eslint/eslint-plugin": ~16.2.0
+    "@angular/cli": ~16.2.0
+    "@angular/common": ~16.2.0
+    "@angular/compiler": ~16.2.0
+    "@angular/compiler-cli": ~16.2.0
+    "@angular/core": ~16.2.0
+    "@angular/platform-browser": ~16.2.0
+    "@angular/platform-browser-dynamic": ~16.2.0
+    "@angular/router": ~16.2.0
+    "@babel/core": ~7.23.0
+    "@babel/preset-typescript": ~7.23.0
+    "@ngrx/entity": ~16.2.0
+    "@ngrx/store": ~16.2.0
+    "@nx/eslint-plugin": ~16.8.0
+    "@nx/jest": ~16.8.0
+    "@nx/js": ~16.8.0
+    "@nx/linter": ~16.8.0
+    "@o3r/build-helpers": "workspace:^"
+    "@o3r/core": "workspace:^"
+    "@o3r/dev-tools": "workspace:^"
+    "@o3r/eslint-plugin": "workspace:^"
+    "@o3r/schematics": "workspace:^"
+    "@o3r/test-helpers": "workspace:^"
+    "@schematics/angular": ~16.2.0
+    "@types/jest": ~29.5.2
+    "@types/minimist": ^1.2.2
+    "@types/node": ^18.0.0
+    "@types/pid-from-port": ^1.1.0
+    "@types/semver": ^7.3.13
+    "@types/uuid": ^9.0.0
+    "@typescript-eslint/eslint-plugin": ^5.60.1
+    "@typescript-eslint/parser": ^5.60.1
+    commander: ^11.0.0
+    cpy-cli: ^4.2.0
+    eslint: ^8.42.0
+    eslint-import-resolver-node: ^0.3.4
+    eslint-plugin-jest: ~27.4.0
+    eslint-plugin-jsdoc: ~46.8.0
+    eslint-plugin-prefer-arrow: ~1.2.3
+    eslint-plugin-unicorn: ^47.0.0
+    globby: ^11.1.0
+    jest: ~29.7.0
+    jest-environment-jsdom: ~29.7.0
+    jest-junit: ~16.0.0
+    jest-preset-angular: ~13.1.1
+    minimist: ^1.2.6
+    ngrx-store-localstorage: ^16.0.0
+    nx: ~16.8.0
+    pid-from-port: ^1.1.3
+    rxjs: ^7.8.1
+    semver: ^7.5.2
+    ts-jest: ~29.1.1
+    tslib: ^2.5.3
+    type-fest: ^3.12.0
+    typescript: ~5.1.6
+    uuid: ^9.0.0
+    winston: ^3.8.2
+    zone.js: ~0.13.1
+  peerDependencies:
+    "@angular-devkit/architect": ~0.1602.0
+    "@angular-devkit/core": ~16.2.0
+    "@angular-devkit/schematics": ~16.2.0
+    "@angular/cli": ~16.2.0
+    "@angular/common": ~16.2.0
+    "@angular/compiler-cli": ~16.2.0
+    "@angular/core": ~16.2.0
+    "@o3r/schematics": "workspace:^"
+    "@schematics/angular": ~16.2.0
+    typescript: ~5.1.6
+  peerDependenciesMeta:
+    "@angular/cli":
+      optional: true
+  bin:
+    o3r-set-version: ./dist/src/cli/set-version.cjs
```

Comment:
You don't update the pipeline to use the new bin?

Reply:
The pipeline is still relying on the one expose by the `@o3r/dev-tools`, not this one.
The purpose behind it is that I don't want to build `@o3r/workspace` in postinstall (because to heavy and with too much deps).
As `@o3r/dev-tools` is deprecated, the choice regarding the run of scripts needed by the `@o3r/framework` mono-repo itself will need to be addressed for all of them (cf. PR on the version-harmonize)